Cython/Utility/ModuleSetupCode.c

```diff
@@ -2915,8 +2915,7 @@ static PyObject *__Pyx_State_FindModule(CYTHON_UNUSED void* dummy) {
 static void __Pyx_ModuleStateLookup_wait_until_no_readers(__Pyx_ModuleStateLookupData *data) {
     // Wait for any readers still working on the old data. Spin-lock is
     // fine because readers should be much faster than memory allocation.
-    // This is called while also being a reader, so allow one reader.
-    while (__pyx_atomic_load(&data->read_counter) > 1);
+    while (__pyx_atomic_load(&data->read_counter) > 0);
```

Comment:
Ok, it's an atomic load, but that doesn't guarantee that _after_ the loop the read counter is _still_ 0. Doesn't that matter?